homeassistant/components/hassio/const.py

```diff
@@ -33,7 +33,8 @@
 WS_TYPE = "type"
 WS_ID = "id"
 
-WS_TYPE_EVENT = "supervisor/event"
 WS_TYPE_API = "supervisor/api"
+WS_TYPE_EVENT = "supervisor/event"
+WS_TYPE_SUBSCRIBE = "supervisor/subscribe"
```

Comment:
For a future PR: since we only use these values inside the schema that describes the command, I have been actually moving away from storing them in constants in other places.